index.js

```diff
@@ -19,7 +19,7 @@ import {
 import PdfViewNativeComponent, {
     Commands as PdfViewCommands,
   } from './fabric/RNPDFPdfNativeComponent';
-import ReactNativeBlobUtil from 'react-native-blob-util'
+import ReactNativeBlobUtil from 'rn-fetch-blob'
```

Comment:
bro, https://github.com/joltup/rn-fetch-blob is already abandoned. last update is on 2019